src/commands/run/spec/run.spec.server.ts

```diff
@@ -201,7 +201,7 @@ describe('sasjs run', () => {
         expect(result.log.includes(logPart)).toBeTruthy()
       },
 
-      60 * 1000
+      5 * 60 * 1000
```

Comment:
Why you've decided to increase the timeout?

Reply:
explained above.

Reply:
explained above

Reply:
fixed